src/components/MeetingsList.js

```diff
@@ -0,0 +1,72 @@
+import { useEffect, useState } from "react"
+import { useParams } from "react-router-dom"
+
+function MeetingsList() {
+  const { id } = useParams()
+  const initialData = { about: "", location: "", time: "", contactId: id }
+  const [meetingData, setMeetingData] = useState(initialData)
+  const [meetings, setMeetings] = useState([])
+
+  useEffect(() => {
+    fetch(`http://localhost:4000/meetings?contactId=${id}`)
+      .then(res => res.json())
+      .then(data => setMeetings(data))
+  }, [id])
```

Comment:
what does the [id] do here?

Reply:
The fetch url depends on 'id' so I added 'id' to the array of things to track, so the function runs everytime 'id' changes.